Sources/FluentKit/Migration/Migration.swift

```diff
@@ -1,6 +1,19 @@
+/// `Migration` can assist with database migration. It can choose
+/// to create a new database schema for migration, or drop an
+/// existing schema. These actions are done in-place, returning no
+/// values.
 public protocol Migration {
     var name: String { get }
```

Comment:
This should be documented as well - it's the name of the migration which Fluent uses to track the state of